doc/releases/changelog-dev.md

```diff
@@ -179,6 +179,12 @@
 
 <h3>Deprecations 👋</h3>
 
+* The boolean functions provided by ``pennylane.operation`` are deprecated. See each function's docstring
```

Comment:
```suggestion
* The boolean functions provided by ``pennylane.operation`` are deprecated. See each function's source code 
```

Reply:
```suggestion
* The boolean functions provided by ``pennylane.operation`` are deprecated. See the deprecations page
```
I really don't think we should be recommending people look at our source code.